components/OutivityDetail/index.js

```diff
@@ -20,33 +27,47 @@ export default function OutivityDetail({ outivity, onDeleteOutivity }) {
       <Head>
         <title>{outivity.title}</title>
       </Head>
-      <main>
-        <article>
-          <ul>
-            <li>
-              <h2>{outivity.title}</h2>
-              <StyledOutivityImage
-                src={outivity.image}
-                alt={outivity.title}
-                width={300}
-                height={200}
-                layout="fixed"
-              />
-              <p>
-                <strong>Location: </strong> {outivity.area},
-                <StyledCountryName>{outivity.country}</StyledCountryName>
-              </p>
+      {!isEdit ? (
+        <main>
+          <article>
+            <ul>
```

Comment:
Why are you using a list here? There is no semantical reason for it.

Reply:
Very good point! We changed this.